components/idp-mgt/org.wso2.carbon.idp.mgt.ui/src/main/resources/web/idpmgt/idp-mgt-edit-oidc.jsp

```diff
@@ -176,9 +182,15 @@
     if (isOIDCBasicAuthEnabled) {
         oidcBasicAuthEnabledChecked = "checked=\'checked\'";
     }
+    if (scopes == null) {
+        scopes = StringUtils.EMPTY;
+    }
     if (oidcQueryParam == null) {
         oidcQueryParam = StringUtils.EMPTY;
     }
+    if (StringUtils.isBlank(scopes) && !oidcQueryParam.toLowerCase().contains("scope=")) {
```

Comment:
Do we have this validation in backend level too? If no scopes are set, are we setting openid scope?